usr/lib/firmware/dsdt/ayaneo_air_plus_D16_0x0F.dsl

```diff
@@ -0,0 +1,9969 @@
+/*
+ * Intel ACPI Component Architecture
+ * AML/ASL+ Disassembler version 20200925 (64-bit version)
+ * Copyright (c) 2000 - 2020 Intel Corporation
+ * 
+ * Disassembling to symbolic ASL+ operators
+ *
+ * Disassembly of dsdt.dat, Fri Aug  4 16:46:29 2023
+ *
+ * Original Table Header:
+ *     Signature        "DSDT"
+ *     Length           0x00008A6C (35436)
+ *     Revision         0x02
+ *     Checksum         0x0F
+ *     OEM ID           "ALASKA"
+ *     OEM Table ID     "A M I "
+ *     OEM Revision     0x01072009 (17244169)
+ *     Compiler ID      "INTL"
+ *     Compiler Version 0x20200717 (538969879)
+ */
+DefinitionBlock ("", "DSDT", 2, "ALASKA", "A M I ", 0x01072009)
```

Comment:
You will need to increment this to 0x0107200a or the system will not load this dsdt.